src/wmnds/patterns/cookies/cookies.js

```diff
@@ -0,0 +1,117 @@
+const cookies = () => {
+  const hideCookieBanner = () => {
+    const cookiesBanner = document.querySelector('.wmnds-cookies-banner');
+    cookiesBanner.style = 'display:none';
+  };
+
+  const setCookie = (cname, cvalue, exdays) => {
+    const d = new Date();
+    d.setTime(d.getTime() + exdays * 24 * 60 * 60 * 1000);
+    const expires = 'expires=' + d.toUTCString(); // eslint-disable-line prefer-template
+    const domain = 'domain=' + window.location.hostname; // eslint-disable-line prefer-template
+    document.cookie = cname + '=' + cvalue + ';' + expires + ';' + domain + ';path=/'; // eslint-disable-line prefer-template
+  };
+
+  const getCookie = cname => {
+    const name = cname + '='; // eslint-disable-line prefer-template
+    const ca = document.cookie.split(';');
+    for (let i = 0; i < ca.length; i += 1) {
+      let c = ca[i];
+      while (c.charAt(0) === ' ') {
+        c = c.substring(1);
+      }
+      if (c.indexOf(name) === 0) {
+        return c.substring(name.length, c.length);
+      }
+    }
+    return '';
+  };
```

Comment:
Can you add some comments to this file to explain what is going on, as it could be confusing for another dev(or your future self) working on this.